message FuncInfo{

	enum Type{
		NONE = 0;
		MISSION = 1;
		BUSINESS = 2;
		ROOM = 3;
	}

	enum ArgSizeType{
		ARG_SIZE = 2;
	}

	optional Type type = 1;
	repeated int32 arg = 2;
}

message FuncAtt{
	enum FuncsSizeType{
		FUNCS_SIZE = 50;
	}

	repeated FuncInfo funcs = 1;
}
